people_tool/templates/people_tool/partials/search.html

```diff
@@ -0,0 +1,94 @@
+{% verbatim %}
+<nav>
+  <h3 class="breadcrumbs">
+    {% endverbatim %}
+    <a href="{% url 'dashboard_account' %}">Admin Console</a>
+    {% verbatim %}
+    <small><i class="fa fa-chevron-right"></i></small>
+    Find People
+  </h3>
+</nav>
+
+<main>
+
+<div class="row" id="search-form-container">
+  <div class="col-xs-offset-2 col-xs-1" id="search-form-label">
+    <label for="search-type-button" class="pull-right">
+      Search by:
+    </label>
+  </div>
+  <div class="col-xs-7" id="search-form-input-group">
+    <form novalidate id="search-form">
+      <div class="form-group">
+        <div class="input-group">
+          <div class="input-group-btn">
+            <button type="button"
+                    class="btn btn-default dropdown-toggle"
+                    id="search-type-button"
+                    data-toggle="dropdown"
+                    ng-bind-html="searchType.name + ' <span class=\'caret\'></span>'"
+                    aria-haspopup="true"
+                    aria-expanded="false">
+            </button>
+            <ul class="dropdown-menu" id="search-type-list">
+              <li ng-repeat="opt in searchTypeOptions">
+                <a href="" ng-click="updateSearchType(opt)"
+                   id="search-type-link-{{opt.key}}">
+                  {{ opt.name }}
+                </a>
+              </li>
+            </ul>
+          </div><!-- /input-group-btn -->
+          <input type="text" class="form-control" id="search-query-string"
```

Comment:
We should consider validating the content and size of the search terms. Since each search term type/backend column def is different, it would have to be a 'soft validation' (a check after submit, instead of limiting the characters in the field itself).